app/components/modal-dialog/component.js

```diff
@@ -8,6 +8,9 @@ export default ModalDialog.extend(EmberKeyboardMixin, {
 	init() {
 		this._super(...arguments)
 		this.set('keyboardActivated', true)
+		// This ensures the modal has first priority to respond to keybard events.
```

Comment:
why does it need to be in `init`?
a property on the component.js, `keyboardActivated: true` would not work?

Reply:
i don't know, it's from http://patience-tema-baron.github.io/ember-keyboard/#/usage. or we can do `{{my-component keyboardActivated=true}}`?